src/lib.rs

```diff
@@ -261,6 +261,30 @@ impl FixedBitSet
             other: other,
         }
     }
+
+    /// In-place union of two `FixedBitSet`s.
+    pub fn union_with(&mut self, other: &FixedBitSet)
+    {
+        if other.len() >= self.len() {
+            self.grow(other.len());
+        }
+        let mn = std::cmp::min(self.data.len(), other.data.len());
+        for i in 0..mn {
+            self.data[i] |= other.data[i];
+        }
+    }
+
+    /// In-place intersection of two `FixedBitSet`s.
+    pub fn intersection_with(&mut self, other: &FixedBitSet)
+    {
+        let mn = std::cmp::min(self.data.len(), other.data.len());
+        for i in 0..mn {
+            self.data[i] &= other.data[i];
+        }
+        for i in mn..self.data.len() {
+            self.data[i] = 0;
+        }
```

Comment:
Definitely for loop for this one?  `for elt in &mut self.data[mn..] { *elt = 0; }`